docs/changelog/88358.yaml

```
@@ -0,0 +1,5 @@
+pr: 88385
+summary: Fixed NullPointerException on bulk request
+area: network
```

Hello @catap, thanks for the fix!
Do you mind updating this value to `Distributed` to make the build pass?

```suggestion
area: Distributed
```